randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -712,14 +712,19 @@ Extra - room_id: [19]
       # Shinespark from adjacent room: https://youtu.be/ID3TOSUmCa8
       Gravity Suit and Level 0 Locks and Speed Booster and Shinespark Tricks (Intermediate) and Disabled Door Lock Rando and Disabled Entrance Rando
   > Room Center
-      Gravity Suit and Screw Attack
+      All of the following:
+          Gravity Suit
+          Any of the following:
+              Screw Attack
+              # TODO: add vid
+              Level 0 Locks and Speed Booster and Shinespark Tricks (Beginner) and Disabled Door Lock Rando and Disabled Entrance Rando
```

Comment:
Either you tested this and then you couldve recorded a video
or you didnt test it and then i want confirmation.